scripts/query/get-manifests.ts

```diff
@@ -0,0 +1,39 @@
+import { detectSnapLocation, fetchSnap } from '@metamask/snaps-controllers';
+import type { SnapId } from '@metamask/snaps-sdk';
+import fs from 'fs';
+import path from 'path';
+import semver from 'semver/preload';
+
+import registry from '../../src/registry.json';
+
+export const MANIFESTS_FILE_LOCATION = path.join(__dirname, 'data.json');
+
+/**
+ * Function to fetch Snaps manifests and write it to a local JSON file.
+ */
+export async function getManifests() {
+  console.log('Fetching Snaps. Please wait ...');
+  const allManifests = [];
+  for (const snap of Object.values(registry.verifiedSnaps)) {
+    console.log(snap.metadata.name);
```

Comment:
Is this log intentional?

Reply:
Yes, It lists the Snaps as it fetches it, so we dont see "no activity" happening while the whole list of snaps are read.